recipes/archicad-apidevkit/all/conanfile.py

```diff
@@ -0,0 +1,36 @@
+from conan import ConanFile
+from conan.tools.files import get
+from conan.errors import ConanInvalidConfiguration
+
+required_conan_version = ">=1.33.00"
+
+
+class ArchicadApidevkitConan (ConanFile):
+    name = "archicad-apidevkit"
+    description = "The General API Development Kit enables software developers to extend the functionality of Archicad"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://archicadapi.graphisoft.com/"
+    license = "https://archicadapi.graphisoft.com/documentation/license-agreement"
+    settings = "os", "arch"
```

Comment:
```suggestion
    settings = "os", "arch", "build_type", "compiler"
```
Keep full settings always, then manage the package ID later